keyboards/tominabox1/littlefoot_lx/keymaps/via/via.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
via json doesn't belong in this repo, and needs to be removed and pr'ed to the-via/keyboards